docs/firebase_layers.js

```diff
@@ -88,13 +89,16 @@ function createContinuousFunction(field, minVal, maxVal, color) {
  */
 function createDiscreteFunction(field, colors) {
   return (feature) => {
-    const color = colors[feature['properties'][field]];
-    const rgba = colorMap.get(color);
+    const value = feature['properties'][field];
+    if (value === null) return transparent;
```

Comment:
I think this is fine, but just FYI, I think you could even just add [null, transparent] as an entry to the map. Javascript apparently allows null keys.